problem_builder/tests/integration/test_mentoring.py

```diff
@@ -442,22 +442,32 @@ def assert_state(answer, mcq, mrq, rating, messages):
 
     @ddt.unpack
     @ddt.data(
-        # MCQ with tips
+        # Questionnaire with tips
         ("feedback_persistence_mcq_tips.xml", '.choice-tips'),
-        # Like the above but instead of tips in MCQ
+        ("feedback_persistence_mrq_tips.xml", '.choice-tips'),
+        # Like the above but instead of tips in MCQ/MRQ
         # has a question level feedback. This feedback should also be suppressed.
-        ("feedback_persistence_mcq_no_tips.xml", '.feedback')
+        ("feedback_persistence_mcq_no_tips.xml", '.feedback'),
+        ("feedback_persistence_mrq_no_tips.xml", '.feedback'),
```

Comment:
@mtyaka I think it would be worth extending this test to also cover cases where an MCQ/MRQ has:

* no question-level feedback and no choice-level feedback (no message displayed when clicking checkmark for individual choice after submitting answer)
* both question-level feedback and choice-level (general message displayed after submitting, choice-level message displayed after clicking corresponding checkmark)

Aside from making the code even more robust against regressions, a test that covers all combinations could also serve to document the expected behavior for us.